VotingApplication/VotingApplication.Web.Api/Controllers/API Controllers/TokenPollVoteController.cs

```diff
@@ -10,12 +10,24 @@
 
 namespace VotingApplication.Web.Api.Controllers.API_Controllers
 {
-    public class UserPollVoteController : WebApiController
+    public class TokenPollVoteController : WebApiController
```

Comment:
It might make more sense for this to be a PollTokenVote controller, to match the PollToken controller ordering